core/templates/dev/head/components/ExplorationSummaryTileMobileDirective.js

```diff
@@ -0,0 +1,140 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Gallery tile component.
+ */
+
+oppia.directive('explorationSummaryTileMobile', [function() {
+  return {
+    restrict: 'E',
+    scope: {
+      getCollectionId: '&collectionId',
+      getExplorationId: '&explorationId',
+      getExplorationTitle: '&explorationTitle',
+      getLastUpdatedMsec: '&lastUpdatedMsec',
+      getNumViews: '&numViews',
+      getObjective: '&objective',
+      getCategory: '&category',
+      getRatings: '&ratings',
+      getContributorsSummary: '&contributorsSummary',
+      getThumbnailIconUrl: '&thumbnailIconUrl',
+      getThumbnailBgColor: '&thumbnailBgColor',
+      // If this is not null, the new exploration opens in a new window when
+      // the summary tile is clicked.
+      openInNewWindow: '@openInNewWindow',
+      isCommunityOwned: '&isCommunityOwned'
+    },
+    templateUrl: 'summaryTile/exploration-mobile',
+    link: function(scope, element) {
```

Comment:
Way too much duplication here (with the non-mobile directive). If we keep this, people will change the code for one of them, and forget to change the other.

Instead, can you add an optional attribute to the existing explorationSummaryTile directive -- something like mobile-cutoff? Then change its HTML template to show the small version if the screen width is below mobile-cutoff, and the large version otherwise. If mobile-cutoff is not specified (as is the case on existing uses of explorationSummaryTile), always show the large version.